lib/experimental/new_embed.dart

```diff
@@ -473,14 +500,23 @@ class DisableableButton {
       : assert(anchorElement != null),
         assert(onClick != null) {
     _element = DElement(anchorElement);
-    _element.onClick.listen((e) => onClick());
+    _element.onClick.listen((e) {
+      if (!_disabled) {
+        onClick();
+      }
+    });
   }
 
   static const disabledClassName = 'disabled';
 
   DElement _element;
+  bool _disabled = false;
```

Comment:
nit: add a whitespace line above this so all the properties are spaced out equally.

Reply:
fixed :)